pulpcore/cli/rpm/context.py

```diff
@@ -102,6 +102,17 @@ class PulpRpmRemoteContext(PulpRemoteContext):
         "sles_auth_token",
     }
 
+class PulpUlnRemoteContext(PulpRemoteContext):
+    ENTITY = _("uln remote")
+    ENTITIES = _("uln remotes")
+    HREF = "rpm_uln_remote_href"
+    ID_PREFIX = "remotes_rpm_uln"
+    NULLABLES = {
```

Comment:
Any reason not to extend the set of the base class here?

Reply:
Extended the RPm Remote base class and fixed Nullables as well.

Reply:
OK, sorry i didn't express this properly. We can still safely inherit from `PulpRemoteContext`, while we would define `NULLABLES = PulpRemoteContext.NULLABLES + {"uln_server-base_url"}`.

Reply:
But the question is, can we have nullable `"uln_server-base_url"`? In settings we set to default value and although its not a required field but we cannot set it to Null. Or am I understanding `Nullables` wrong?

Reply:
OK that i don't know. `NULLABLES` is basically to translate the empty string provided if you specify `--opt ""` to `None` for submitting to the api.
Whether `uln_server_base_url` can be "unset" is a question for someone else. If it is not, i'd say maybe you don't need to redefine `NULLABLES` at all...
Also think about the update command. Maybe you need to provide a version of that option without default, or it will unintentionally be overwritten.